src/schemas/json/bitrise.json

```diff
@@ -0,0 +1,445 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$ref": "#/definitions/BitriseDataModel",
+  "definitions": {
+    "AppModel": {
+      "properties": {
+        "title": {
+          "type": "string"
+        },
+        "summary": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "envs": {
+          "items": {
+            "patternProperties": {
+              ".*": {
+                "additionalProperties": true
+              }
+            },
+            "type": "object"
+          },
+          "type": "array"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "AptGetDepModel": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "bin_name": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "BashStepToolkitModel": {
+      "properties": {
+        "entry_file": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "BitriseDataModel": {
+      "required": ["format_version", "project_type"],
```

Comment:
project_type is required? 🤔

Reply:
It is required in the model, interestingly. Wdyt, should we change the models too?